client/lib/src/process_event.rs

```diff
@@ -19,38 +17,60 @@ pub fn process_event<UpdateServer>(
   view: &mut view::T,
   client: &client::T,
   event: Event,
+  key_repeated: &mut bool,
```

Comment:
This is buggy when more than one key is pressed. Does glutin not have first-class support for distinguishing repeated keys? It seems like there would be edge cases around window focus too.

Reply:
Glutin doesnt have first-class support for repeated keys.